app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def only_host_and_path(input_url)
+    if input_url.present?
+      url = URI(input_url)
+
+      link_to url.host + url.path, input_url
+    else
+      nil
+    end
+  end
```

Comment:
Is this helper method designed to remove the query and any fragment from the input URL (only leaving the host and path)? If so, is this an aesthetic choice mostly or is there a hardening aspect to it (other than the optics of removing it from the visible URL)?

Reply:
Good question. This is purely an aesthetic choice, since we're still providing the user-entered URL as the href of the link.